README.md

```diff
@@ -1,2 +1,47 @@
 # WeatherApp-Js
-Practice API with Js Promises and Async/Await while Manipulating the DOM.
+
+> Practice API with Js Promises and Async/Await while Manipulating the DOM.
+
+# Built With
+
+- HTML
+- CSS
+- JavaScript
+
+# Demo Link
+Visit [Link](https://fr3elancer.github.io/WeatherApp-Js/)
+
+# Getting started
+
+## To get a local copy of this repository kindly follow the steps below.
+- Scroll to top of this current repository
+- Click on the `Code` button with background color green on the right end corner
+- Click on the clipboard icon on the extreme right of the dropdown to copy the repository link
+- In your local PC, open your terminal or command prompt in the folder you would like to clone this repository into
+- Type `git clone (copied link)` on the currently opened terminal or command prompt
+
+# Getting Started 
+
+- Open your Terminal and Navigate to the project destination and type `npm install`
+- Run `npm run build`
+- You can now open the `index.html` file at `dist` folder in any browser.
```

Comment:
Great 👍 

But we are using webpack, so let's use the `npm start` command to start up the app